utils/build_runtime_tests.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2020 Google Inc. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+if [ -z $1 ]; then
+  COMPILATION_LEVEL="SIMPLE"
+else
+  COMPILATION_LEVEL=$1
+fi
+
+# to translate from relative dir
+abs_dirname() {
+  echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
+}
+
+LOCAL_COMPILER=$(dirname ..)/target/closure-compiler-1.0-SNAPSHOT.jar
+if [ ! -f "$LOCAL_COMPILER" ]; then
+  echo -e "\nCompiler JAR not built. Building...\n" && yarn build:fast
+fi
+
+echo -e "\nBuilding runtime tests..."
+TEST_DIR="test/com/google/javascript/jscomp/runtime_tests"
+
+if [ -z $1 ]; then
+  ABS_PATH=$(abs_dirname "./$TEST_DIR")
+else
+  ABS_PATH=$(abs_dirname "$1")
+fi
+
+i=0
+compileRuntimeTests(){
+  for FILE in $@; do
+
+    FILE_BASE=$(echo $FILE | rev | cut -f 2- -d '.' | rev)
+    TEST_NAME=$(basename $FILE_BASE)
+    TEST_LOC=$(dirname $FILE)
```

Comment:
It's common to just assign to variable names like this in shell scripts, but it is actually better style in a function to declare
them to be local, so they don't pollute the global namespace. Sadly all variables are global in bash unless explicitly declared otherwise.

```sh
local testName=$(basename $fileBase)
```